vulnerabilities/api.py

```diff
@@ -254,6 +254,14 @@ def bulk_search(self, request):
 
         return Response(response)
 
+    @action(detail=False, methods=["get"])
+    def all(self, request):
+        vulnerable_packages = Package.objects.filter(
+            packagerelatedvulnerability__fix=False
+        ).distinct()
```

Comment:
We could add `.only(*PackageURL._fields)` to get the minimal data required from the DB to build the purls.

Also, `.filter(packagerelatedvulnerability__fix=False)` should be a method on the QuerySet since it's used in multiple places. It would improve code readability by using a simple method name.

Reply:
See also https://github.com/nexB/vulnerablecode/issues/880

Reply:
Done!